currency.worker.js

```diff
@@ -0,0 +1,37 @@
+const axios = require('axios');
+const Agenda = require('agenda');
+require('dotenv').config();
+const currencyService = require('./modules/currency/currency.service');
+
+const currencyWorker = () => {
+  const agenda = new Agenda({ db: { address: process.env.MONGO_URL } });
+  agenda.define('set currency to database', async job => {
+    try {
+      const currency = await axios.get(process.env.CURRENCY_API_URL);
+
+      const currencyRate = {
+        lastUpdatedDate: Date.now(),
+        convertOptions: [
+          {
+            name: currencyService.currencyTypes[0],
+            exchangeRate: currency.data.rates.UAH,
+          },
+          {
+            name: currencyService.currencyTypes[1],
```

Comment:
It is a wrong approach, use an object with keys instead, like `CurrencyTypes.USD`, also, please, export this as a separate variable from the `currency.service.js`, not as a part of the same service

Reply:
fixed